ingestion/src/metadata/ingestion/source/database/sample_data.py

```diff
@@ -1426,7 +1426,10 @@ def ingest_test_case(self) -> Iterable[Either[OMetaTestCaseSample]]:
                             )
                             create_test_case_resolution.testCaseResolutionStatusDetails = Assigned(
                                 assignee=EntityReference(
-                                    id=user.id.__root__, type="user"
+                                    id=user.id.__root__,
```

Comment:
assuming entity can be team or user?
also lets make sure we are validating on the backend side

Reply:
I actually just added support for user as far as task flow creation goes.